packages/sui-studio/src/components/navigation/index.js

```diff
@@ -13,90 +13,83 @@ const compliant = getStudioCompliant()
 const SEARCH_BY_COMPLIANT = '*'
 
 export default class Navigation extends Component {
-  state = { search: '' }
+  state = {search: ''}
 
-  _handleChange = (e) => {
-    this.setState({ search: e.target.value })
+  _handleChange = e => {
+    this.setState({search: e.target.value})
   }
 
-  _handleFocus = (e) => {
+  _handleFocus = e => {
```

Comment:
IMO this is like the one liner ifs without brackets, it will always be annoying to add a new param